modules/executive/components/DateFilter.tsx

```diff
@@ -50,7 +50,7 @@ export default function DateFilter(props): JSX.Element {
       {...props}
     >
       <Grid gap={2} columns="max-content max-content" sx={{ alignItems: 'baseline' }}>
-        <Text>After</Text>
```

Comment:
Slightly tangential, up on line 45, should we change `Date Posted` to `Date Filter` to be consistent with the polling page?